src/codec_dav1d.c

```diff
@@ -57,8 +57,8 @@ static avifBool dav1dCodecGetNextImage(struct avifCodec * codec,
 {
     if (codec->internal->dav1dContext == NULL) {
         // Give all available threads to decode a single frame as fast as possible
-        codec->internal->dav1dSettings.n_frame_threads = 1;
-        codec->internal->dav1dSettings.n_tile_threads = AVIF_CLAMP(decoder->maxThreads, 1, DAV1D_MAX_TILE_THREADS);
+        codec->internal->dav1dSettings.n_threads = 1;
+        codec->internal->dav1dSettings.max_frame_delay = AVIF_CLAMP(decoder->maxThreads, 1, 256);
```

Comment:
Perhaps it's better to use `DAV1D_MAX_THREADS`  instead of a bare number

Reply:
`DAV1D_MAX_THREADS` is delete in david 0.9.2+. I guess that changed the value from 64 to 256.

Reply:
I see it at https://code.videolan.org/videolan/dav1d/-/merge_requests/1252/diffs#95378459063068eb66ef004414047378fc28a963_49_46 and at https://code.videolan.org/videolan/dav1d/-/blob/master/include/dav1d/dav1d.h#L46

Reply:
Wrong macro